web/docs/language/features.md

```diff
@@ -1173,6 +1181,30 @@ entity SocialLogin {=psl
 psl=}
 ```
 
+## Compiler configuration
+
+You can configure Wasp's compiler using the `wasp` field inside the `app`
+declaration.
+
+```c
+app MyApp {
+  wasp: {
+    version: "^0.6.0"
+  },
+  // ...
+}
+```
+
+`app.wasp` is a dictionary with the following fields:
+
+#### `version: string` (required)
+
+`version` declares the compatible Wasp versions for the app. It is specified
+in a SemVer format.
+
+**Note**: For now, `version` only supports the format `^0.major.minor.patch`. This
```

Comment:
Let's update this one to be `^0.y.z`.

Reply:
As the `SemanticVersion` module handles the `0.y.z` case correctly, I think it's okay to leave it as `^x.y.z` instead. What do you think?

Reply:
Yup, good point, forgot about that! Let's do it!